internal/testutil/acc/attribute_checks_test.go

```diff
@@ -56,3 +57,14 @@ func TestJSONEquals(t *testing.T) {
 		})
 	}
 }
+
+func TestAddNoAttrSetChecks(t *testing.T) {
+	asserter := assert.New(t)
```

Comment:
knit: i prefer assert := ..., one the few exceptions I think it's ok to shadow, then assert.Empty, etc.

Reply:
Hmm. I couldn't find this practice in other places in the code base. (searching for `assert := assert.New(t)` )
And the `golangci-lint run` will complain about it:

```
internal/testutil/acc/attribute_checks_test.go:62:2: importShadow: shadow of imported from 'github.com/stretchr/testify/assert' package 'assert' (gocritic)
	assert := assert.New(t)
	^
```
Therefore, I prefer keeping it as is